flask_core/app.py

```diff
@@ -39,4 +55,8 @@ def create_app(config=None):
     # Register all our middleware
     app.wsgi_app = Handler(app.wsgi_app)
 
+    # Set up magic flag generator (ty closure)
+    app.all_flag_ids = app.config["FLAG_IDS"].split(",")
```

Comment:
Why is this here? This is still a config item so it should still live under `app.config`. Run the split in config instead.